src/commons/BrowserUtils.ts

```diff
@@ -805,95 +741,78 @@ export namespace BrowserUtils {
   }
 
   /**
-   *
-   * @param selector - element for get size
+   * Get the width and height for an DOM-element.
+   * @param selector requested element selector
    */
-  export function getElementSize(selector: string): Size {
+  export function getSize(selector: string): Size {
     Reporter.debug(`Get Element: '${selector}' size`);
     waitForDisplayed(selector);
 
     return $(selector).getSize();
   }
 
   /**
-   * Change size of browser window
+   * Resizes browser window outer size according to provided width and height.
    * @param width - Width (px)
    * @param height - Height (px)
    */
   export function setWindowSize(width: number, height: number): void {
     Reporter.debug(`Set window size to '${width}X${height}'`);
-    switch (browser.capabilities['browserName']) {
-      case 'chrome': {
-        tryBlock(() => browser.setWindowSize(width, height), 'Chrome: Failed to resize window');
-        break;
-      }
-
-      case 'firefox': {
-        tryBlock(() => browser.setWindowRect(0, 0, width, height), 'FireFox: Failed to resize window');
-        break;
-      }
-
-      default: {
-        throw new TypeError('Unable to execute due to unsupported Browser');
-      }
-    }
+    tryBlock(() => browser.setWindowSize(width, height), 'Chrome: Failed to resize window');
   }
 
+  /**
+   * Returns browser window size.
+   * output example `{ width: 1280, height: 767 }`
+   */
   export function getWindowSize(): object {
```

Comment:
maybe better to return the interface?

Reply:
the BrowserSize is not exported. We can create a PR in  wdio repo with fix